### PR TITLE
STOR-1432: hypershift: pass through control plane images to AWS EBS CSI driver operator

### PR DESCRIPTION
**Pull request by sjenning**
continuing work of
* https://github.com/openshift/cluster-storage-operator/pull/392
* https://github.com/openshift/hypershift/pull/2882

with a follow on in https://github.com/openshift/aws-ebs-csi-driver-operator

https://github.com/openshift/aws-ebs-csi-driver-operator/pull/252

### CODE REVIEW

**Comment by sjenning** [comment]
cc @jsafrane @gnufied @bertinatto

File: pkg/operator/csidriveroperator/csioperatorclient/aws.go [hunk 1]
```diff
@@ -8,15 +8,17 @@ import (
 )
 
 const (
-	AWSEBSCSIDriverName          = "ebs.csi.aws.com"
-	envAWSEBSDriverOperatorImage = "AWS_EBS_DRIVER_OPERATOR_IMAGE"
-	envAWSEBSDriverImage         = "AWS_EBS_DRIVER_IMAGE"
+	AWSEBSCSIDriverName              = "ebs.csi.aws.com"
+	envAWSEBSDriverOperatorImage     = "AWS_EBS_DRIVER_OPERATOR_IMAGE"
+	envAWSEBSDriverImage             = "AWS_EBS_DRIVER_IMAGE"
+	envAWSEBSDriverControlPlaneImage = "AWS_EBS_DRIVER_CONTROL_PLANE_IMAGE"
```
Comment (gnufied) [inline pkg/operator/csidriveroperator/csioperatorclient/aws.go:14]: Very tiny nit: Can you put this on a new line so as this constant stands out apart from others?

File: pkg/operator/csidriveroperator/util.go [hunk 2]
```diff
@@ -18,13 +18,14 @@ import (
...
-	envNodeDriverRegistrarImage = "NODE_DRIVER_REGISTRAR_IMAGE"
-	envLivenessProbeImage       = "LIVENESS_PROBE_IMAGE"
-	envKubeRBACProxyImage       = "KUBE_RBAC_PROXY_IMAGE"
+	envProvisionerImage               = "PROVISIONER_IMAGE"
+	envAttacherImage                  = "ATTACHER_IMAGE"
+	envResizerImage                   = "RESIZER_IMAGE"
+	envSnapshotterImage               = "SNAPSHOTTER_IMAGE"
+	envNodeDriverRegistrarImage       = "NODE_DRIVER_REGISTRAR_IMAGE"
+	envLivenessProbeImage             = "LIVENESS_PROBE_IMAGE"
+	envLivenessProbeControlPlaneImage = "LIVENESS_PROBE_CONTROL_PLANE_IMAGE"
```
Comment (gnufied) [inline pkg/operator/csidriveroperator/util.go:27]: same as above - it is easy to miss them when casually scanning the list.

**Pull request merged**